Automattic/wp-calypso
# Remove Twitter in Calypso on the Marketing page (Tools → Marketing)

## Proposed Changes

* Replace Twitter icon with Mastodon on Marketing page

## Testing Instructions



* Boot up this PR or use the Live links below
* Goto `/marketing/tools/:site`
* Scroll down to Social Sharing section
* Confirm that Twitter icon is replaced with Mastodon icon
* Update Tumblr icon color to the correct brand color

## Pre-merge Checklist




| BEFORE | AFTER |
|--------|--------|
| <img width="231" alt="Screenshot 2024-04-18 at 7 38 11 PM" src="https://github.com/Automattic/wp-calypso/assets/18226415/6c1bcfcd-7c03-4bbf-9a07-4bd1656879d3"> | <img width="230" alt="Screenshot 2024-04-18 at 7 38 22 PM" src="https://github.com/Automattic/wp-calypso/assets/18226415/021b2d7d-2cad-4862-93c4-83a020697feb"> |
